src/screens/application/container/applicationContainer.tsx

```diff
@@ -247,13 +248,29 @@ class ApplicationContainer extends Component {
             {
               text: intl.formatMessage({ id: 'alert.update' }),
               onPress: () => {
-                setLastUpdateCheck(null);
-                Linking.openURL(
-                  Platform.select({
-                    ios: 'itms-apps://itunes.apple.com/us/app/apple-store/id1451896376?mt=8',
-                    android: 'market://details?id=app.esteem.mobile.android',
-                  }),
-                );
+                DeviceInfo.getInstallerPackageName().then((installerPackageName) => {
+                  switch (installerPackageName) {
+                    case 'google-store':
```

Comment:
installerPackageName values used in switch statement will always fail... please revisit `getInstallerPackageName` documentation here....

https://www.npmjs.com/package/react-native-device-info#getinstallerpackagename


For example... in case of android play store, the installer package name would be `com.android.vending` and for apple app store it is `AppStore`